apps/web/src/components/Composer/Actions/Attachment.tsx

```diff
@@ -60,14 +36,18 @@ const Attachment: FC = () => {
     return false;
   };
 
-  const isImageType = (files: FileList) => {
-    for (const file of files) {
-      if (!ALLOWED_IMAGE_TYPES.includes(file.type)) {
-        return false;
-      }
+  const isUploadAllowed = (files: FileList) => {
+    if (files && files[0].type.slice(0, 5) === 'image') {
```

Comment:
Change `files && files[0]` to `files?.[0]`

Reply:
Will sliced type always has 5 chars?

Reply:
Also where we are checking for `ALLOWED_VIDEO_TYPES`?

Reply:
Sliced type always has 5 chars:
video, audio or image

Reply:
> Also where we are checking for `ALLOWED_VIDEO_TYPES`?

There is a function called is Allowed types which confirms whether the type uploaded is permitted.

https://github.com/lensterxyz/lenster/blob/47209cf476c4f34cdd46bfb23d427a846b048c43/apps/web/src/components/Composer/Actions/Attachment.tsx#L63